packages/babel-parser/src/parser/comments.js

```diff
@@ -38,6 +38,48 @@ export default class CommentsParser extends BaseParser {
     this.state.leadingComments.push(comment);
   }
 
+  adjustCommentsAfterTrailingComma_(node: Node, elements: Node[]) {
```

Comment:
Why the trailing underscore?

Reply:
Oh I wanted to mark it as a private function and out of habit followed the convention from another repo. I don't see a convention in babel so I'll just drop the underscore.